src/mesh/gettingstarted.md

```diff
@@ -25,16 +24,15 @@ Download and install {{site.mesh_product_name}} from the
 
 ## 2. Getting Started
 
-After you install, follow the Kuma getting started guide to get
+After you install, follow the getting started guides to get
 {{site.mesh_product_name}} up and running:
 
-* [Getting started with Kubernetes](https://kuma.io/docs/latest/quickstart/kubernetes/){:target="_blank"}
-* [Getting started with Universal](https://kuma.io/docs/latest/quickstart/universal/){:target="_blank"}
+* [Getting started with Kubernetes](/mesh/{{page.kong_version}}/quickstart/kubernetes/){:target="_blank"}
+* [Getting started with Universal](/mesh/{{page.kong_version}}/quickstart/universal/){:target="_blank"}
 
 ## 3. Learn more
 
-* Read the [Kuma documentation](https://kuma.io/docs/){:target="_blank"}
-* Learn about enterprise features:
+Learn about enterprise features:
   * [Support for HashiCorp Vault CA](/mesh/{{page.kong_version}}/features/vault)
   * [Support for Amazon Certificate Manager Private CA](/mesh/{{page.kong_version}}/features/acmpca)
   * [Support for Kubernetes cert-manager CA](/mesh/{{page.kong_version}}/features/cert-manager)
```

Comment:
```suggestion
  * [Support for Kubernetes cert-manager CA](/mesh/{{page.kong_version}}/features/cert-manager)
 
<!-- links -->
{% if_version gte:2.0.x %}
[get-started-k8s]: (/mesh/{{page.kong_version}}/quickstart/kubernetes/)
[get-started-universal]: (/mesh/{{page.kong_version}}/quickstart/universal/)
{% endif_version %}

{% if_version lte:1.9.x %}
[get-started-k8s]: https://kuma.io/docs/latest/quickstart/kubernetes/
[get-started-universal]: https://kuma.io/docs/latest/quickstart/universal/
{% endif_version %}
```
This should work and fix the broken link issue. 

The broken /install/ link is fixed on the release branch, so you can ignore it for now.